base/reflection.jl

```diff
@@ -842,9 +849,9 @@ end
 """
     which(symbol)
 
-Return the module in which the binding for the variable referenced by `symbol` was created.
+Return the module in which the binding for the variable referenced by Main.`symbol` was created.
```

Comment:
As I wrote above, `current_module` returned `Main`, regardless of where it was called from. Yes, this was highly confusing.